examples/org-viewer-react/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
+  .BundleAnalyzerPlugin
+
+module.exports = {
+  entry: './src/index.tsx',
+  devtool: 'inline-source-map',
+  target: 'web',
+  mode: process.env.NODE_ENV || 'development',
+  stats: {
+    preset: process.env.NODE_ENV === 'production' ? 'errors-only' : 'normal',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        exclude: /node_modules/,
+        use: [
+          {
+            loader: 'ts-loader',
+            options: { projectReferences: true },
+          },
+        ],
+      },
+    ],
+  },
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+    mainFields: ['module', 'browser', 'main'],
+  },
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devServer: {
+    contentBase: path.join(__dirname, 'dist'),
+    port: 1234,
+  },
+  plugins: [
+    ...(process.env.BUNDLE_STATS === '1' ? [new BundleAnalyzerPlugin()] : []),
```

Comment:
Btw, how are we doing with this? What's this example clocking in at?

Reply:
1.01 MB for this demo in total, 59.23% of which is ethers.js.

![image](https://user-images.githubusercontent.com/36158/86543402-fe4e8a80-bf15-11ea-8a99-93ec76477bae.png)